src/app/reskin/withdraw/hooks/use-withdrawal-info.ts

```diff
@@ -27,5 +27,6 @@ export function useWithdrawalInfo({ txid }: { txid: string }) {
       }
       return 5000;
     },
+    enabled: !!txid,
```

Comment:
Ahhh, this fixes the issue, since this only executes the query when the txid exists. So now it waits until it gets that from the wallet?